ee/codegen/src/__test__/nodes/__snapshots__/tool-calling-node.test.ts.snap

```diff
@@ -0,0 +1,125 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`ToolCallingNode > basic > getNodeDisplayFile 1`] = `
+"from uuid import UUID
+
+from vellum_ee.workflows.display.editor import NodeDisplayData, NodeDisplayPosition
+from vellum_ee.workflows.display.nodes import BaseNodeDisplay
+from vellum_ee.workflows.display.nodes.types import PortDisplayOverrides
+
+from ...nodes.tool_calling_node import ToolCallingNode
+
+
+class ToolCallingNodeDisplay(BaseNodeDisplay[ToolCallingNode]):
+    attribute_ids_by_name = {
+        "ml_model": UUID("75bd1347-dca2-4cba-b0b0-a20a2923ebcc"),
+        "blocks": UUID("beec5344-2eff-47d2-b920-b90367370d79"),
+        "functions": UUID("7b1ab802-3228-43b3-a493-734c94794710"),
+        "prompt_inputs": UUID("38cf126e-a186-4a63-8e30-47c4507413cd"),
+        "max_tool_calls": UUID("723f614a-be30-4f27-90d0-896c740e58d3"),
+    }
+    port_displays = {
+        ToolCallingNode.Ports.default_port: PortDisplayOverrides(
+            id=UUID("2544f9e4-d6e6-4475-b6a9-13393115d77c")
+        )
+    }
+    display_data = NodeDisplayData(
+        position=NodeDisplayPosition(x=0, y=0), width=None, height=None
+    )
+"
+`;
+
+exports[`ToolCallingNode > basic > getNodeFile 1`] = `
+"from vellum.workflows.nodes.experimental.tool_calling_node import (
+    ToolCallingNode as BaseToolCallingNode,
+)
+
+
+class ToolCallingNode(BaseToolCallingNode):
+    ml_model = "gpt-4o-mini"
+    blocks = [
+        {
+            "state": None,
```

Comment:
[oos] The next iteration we'll want here is to generate the prompt block dataclasses. This may require revisiting the last part of that doc we discussed last week

Reply:
However, I would _still_ expect the workflow to be runnable with the dict's instead. If they aren't we should tackle that first